sys/php/www.conf

```diff
@@ -12,16 +12,16 @@
 ; - 'chdir'
 ; - 'php_values'
 ; - 'php_admin_values'
-; When not set, the global prefix (or NONE) applies instead.
+; When not set, the global prefix (or /opt/bref) applies instead.
 ; Note: This directive can also be relative to the global prefix.
 ; Default Value: none
 ;prefix = /path/to/pools/$pool
 
 ; Unix user/group of processes
 ; Note: The user is mandatory. If the group is not set, the default user's group
 ;       will be used.
-user = www-data
-group = www-data
+user = nobody
+group = nobody
```

Comment:
sure?

Reply:
I took the configuration from the bref image